packages/story-editor/src/components/autoSaveCheck/autoSaveDialog.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { __ } from '@googleforcreators/i18n';
+import {
+  Text,
+  THEME_CONSTANTS,
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+} from '@googleforcreators/design-system';
+/**
+ * Internal dependencies
+ */
+import Dialog from '../dialog';
+
+export const AutoSaveDialog = ({ onClose, isOpen, autoSaveLink }) => {
+  return (
+    <Dialog
+      isOpen={isOpen}
+      title={__('AutoSave', 'web-stories')}
+      contentLabel={__('AutoSave Available', 'web-stories')}
+      secondaryText={__('Dismiss', 'web-stories')}
+      onClose={onClose}
+      actions={
+        <>
+          <Button
+            type={BUTTON_TYPES.TERTIARY}
+            size={BUTTON_SIZES.SMALL}
+            onClick={() => onClose()}
+          >
+            {__('Dismiss', 'web-stories')}
+          </Button>
+          <Button
+            type={BUTTON_TYPES.PRIMARY}
+            size={BUTTON_SIZES.SMALL}
+            href={autoSaveLink}
```

Comment:
<!-- potential concern, ask for clarification or investigation -->
**❓question:**

Is it intended that we display the comparison as WordPress does instead of showing the autosaved story itself? Wondering if the autosave comparison is really useful for the user. Or is this just temporarily like this? Thoughts?

Reply:
Great question.

For now (as far as this ticket is concerned) yes.

WordPress doesn't make it super easy to adjust the comparison view and it might not be trivial to iframe the stories there. But it would definitely make for a nice enhancement.

I will create a new ticket for this.